src/components/shared/Header/Header.style.ts

```diff
@@ -0,0 +1,132 @@
+import styled from '@emotion/styled';
+
+export const HeaderBackground = styled.div`
+  position: fixed;
+  top: 0;
+  z-index: 888;
+  height: 2.5rem;
+  max-width: 375px;
+  width: 100%;
+  background-color: #ffffff;
+`;
```

Comment:
css 속성을 조금 더 추가하면 디자인과 똑같은 모습으로 보일 것 같아요!
그리고 max-width는 불필요하다고 생각합니다.

```suggestion
export const HeaderBackground = styled.div`
  position: fixed;
  top: 0;
  left: 0;
  padding: 0 16px;
  z-index: 888;
  height: 2.5rem;
  max-width: 375px;
  width: 100%;
  background-color: #ffffff;
`;
```